### PR TITLE
Create Gourmet 翻译提交

### PR DESCRIPTION
**Pull request by SouthWindmh**
作者已阅读贡献指南并同意 CC 协议。

由 CFPA-Helper 提交。

### CODE REVIEW

**Comment by Cyl18** [comment]
询问一下：你对PR提交器有什么建议吗？有什么不好用的地方吗？

**Comment by SouthWindmh** [comment]
有时候选邮箱时会发生列表中有邮箱信息提交时却显示需选择邮箱的情况



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年8月6日(周日) 上午10:42
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] Create Gourmet 翻译提交 (PR #3624)




 
询问一下：你对PR提交器有什么建议吗？有什么不好用的地方吗？

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  "item.gourmet.ice_cream_scoop": "冰淇淋球",
+  "item.gourmet.chocolate_ice_cream_scoop": "巧克力冰淇淋球",
+  "item.gourmet.sweet_berry_ice_cream_scoop": "甜浆果冰淇淋球",
+  "item.gourmet.ice_cream_cone": "冰淇淋球甜筒",
+  "item.gourmet.chocolate_ice_cream_cone": "巧克力冰淇淋甜筒",
+  "item.gourmet.sweet_berry_ice_cream_cone": "甜浆果冰淇淋甜筒",
+  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒",
+
+  "item.gourmet.carrot_cake": "胡萝卜蛋糕",
+  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底",
```
Comment (seaseii) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:33]: 有点变扭，一般都是说`蛋糕胚`吧

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  "item.gourmet.waffle_stack": "华夫饼塔",
+  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）",
+  "item.gourmet.french_toast": "法国吐司",
+  "item.gourmet.fried_egg": "煎蛋",
+
+  "item.gourmet.incomplete_burger": "汉堡（半成品）",
+  "item.gourmet.bacon_burger": "培根汉堡",
+  "item.gourmet.big_bacon_burger_meal": "大号培根汉堡套餐",
+  "item.gourmet.beef_patty": "牛肉饼",
+  "item.gourmet.ground_beef": "绞牛肉",
```
Comment (seaseii) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:55]:
```suggestion
  "item.gourmet.ground_beef": "牛绞肉",

```

牛身上一个部位的肉

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  "item.gourmet.flatcake": "美式松饼",
+  "item.gourmet.incomplete_flatcake_stack": "美式松饼塔（半成品）",
+  "item.gourmet.flatcake_stack": "美式松饼塔",
+  "item.gourmet.flatcake_batter": "美式松饼糊",
+  "item.gourmet.waffle": "华夫饼",
+  "item.gourmet.waffle_batter": "华夫饼糊",
+  "item.gourmet.waffle_stack": "华夫饼塔",
+  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）",
+  "item.gourmet.french_toast": "法国吐司",
```
Comment (seaseii) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:48]:
```suggestion
  "item.gourmet.french_toast": "法式吐司",

```

优雅点（

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  "item.gourmet.incomplete_burger": "汉堡（半成品）",
+  "item.gourmet.bacon_burger": "培根汉堡",
+  "item.gourmet.big_bacon_burger_meal": "大号培根汉堡套餐",
+  "item.gourmet.beef_patty": "牛肉饼",
+  "item.gourmet.ground_beef": "绞牛肉",
+  "item.gourmet.buns": "小圆面包",
+  "item.gourmet.ice_cube": "冰块",
+  "item.gourmet.carrot_slices": "胡萝卜片",
+
+  "fluid_type.gourmet.seed_oil": "籽油",
```
Comment (seaseii) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:60]:
```suggestion
  "fluid_type.gourmet.seed_oil": "种子油",

```

```suggestion
  "fluid_type.gourmet.seed_oil": "植物油",

```

籽油还真没听过，叫种子油的多（GTCE），植物油的也有点

**Comment by SouthWindmh** [comment]
其实我本来是想要翻译成法兰西吐司，但维基百科告诉我这玩意用的最多的名称是法国吐司。
蛋糕基底倒是我没注意到create也是用烈焰蛋糕胚来翻译。
牛绞肉主要是确实菜市场去的少(&nbsp;
我看到原文动宾就想当然翻译成绞牛肉了。
籽油主要是第一眼想翻译成菜籽油，但发现这是油菜专属就简单翻译成籽油了

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年8月7日(周一) 下午3:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] Create Gourmet 翻译提交 (PR #3624)




 
@seaseii commented on this pull request.
 
 
In projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:
 &gt; +  "item.gourmet.cheese_danish_dough": "奶酪丹麦酥面团", + +  "item.gourmet.dumpling_wrapper": "包子皮", +  "item.gourmet.incomplete_dumpling": "包子（半成品）", +  "item.gourmet.dumpling": "包子", + +  "item.gourmet.ice_cream_scoop": "冰淇淋球", +  "item.gourmet.chocolate_ice_cream_scoop": "巧克力冰淇淋球", +  "item.gourmet.sweet_berry_ice_cream_scoop": "甜浆果冰淇淋球", +  "item.gourmet.ice_cream_cone": "冰淇淋球甜筒", +  "item.gourmet.chocolate_ice_cream_cone": "巧克力冰淇淋甜筒", +  "item.gourmet.sweet_berry_ice_cream_cone": "甜浆果冰淇淋甜筒", +  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒", + +  "item.gourmet.carrot_cake": "胡萝卜蛋糕", +  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底",  
有点变扭，一般都是说蛋糕胚吧
 
 
In projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:
 &gt; +  "item.gourmet.flatcake": "美式松饼", +  "item.gourmet.incomplete_flatcake_stack": "美式松饼塔（半成品）", +  "item.gourmet.flatcake_stack": "美式松饼塔", +  "item.gourmet.flatcake_batter": "美式松饼糊", +  "item.gourmet.waffle": "华夫饼", +  "item.gourmet.waffle_batter": "华夫饼糊", +  "item.gourmet.waffle_stack": "华夫饼塔", +  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）", +  "item.gourmet.french_toast": "法国吐司", +  "item.gourmet.fried_egg": "煎蛋", + +  "item.gourmet.incomplete_burger": "汉堡（半成品）", +  "item.gourmet.bacon_burger": "培根汉堡", +  "item.gourmet.big_bacon_burger_meal": "大号培根汉堡套餐", +  "item.gourmet.beef_patty": "牛肉饼", +  "item.gourmet.ground_beef": "绞牛肉",  ⬇️ Suggested change -  "item.gourmet.ground_beef": "绞牛肉", +  "item.gourmet.ground_beef": "牛绞肉",  
牛身上一个部位的肉
 
 
In projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:
 &gt; +  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底", +  "item.gourmet.incomplete_carrot_cake": "胡萝卜蛋糕（半成品）", + +  "item.gourmet.variety_dessert_platter": "缤纷甜品拼盘", +  "item.gourmet.comforting_dumpling_meal": "温馨包子餐", +  "item.gourmet.southern_breakfast": "南式早餐", + +  "item.gourmet.flatcake": "美式松饼", +  "item.gourmet.incomplete_flatcake_stack": "美式松饼塔（半成品）", +  "item.gourmet.flatcake_stack": "美式松饼塔", +  "item.gourmet.flatcake_batter": "美式松饼糊", +  "item.gourmet.waffle": "华夫饼", +  "item.gourmet.waffle_batter": "华夫饼糊", +  "item.gourmet.waffle_stack": "华夫饼塔", +  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）", +  "item.gourmet.french_toast": "法国吐司",  ⬇️ Suggested change -  "item.gourmet.french_toast": "法国吐司", +  "item.gourmet.french_toast": "法式吐司",  
优雅点（
 
 
In projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:
 &gt; +  "item.gourmet.waffle_batter": "华夫饼糊", +  "item.gourmet.waffle_stack": "华夫饼塔", +  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）", +  "item.gourmet.french_toast": "法国吐司", +  "item.gourmet.fried_egg": "煎蛋", + +  "item.gourmet.incomplete_burger": "汉堡（半成品）", +  "item.gourmet.bacon_burger": "培根汉堡", +  "item.gourmet.big_bacon_burger_meal": "大号培根汉堡套餐", +  "item.gourmet.beef_patty": "牛肉饼", +  "item.gourmet.ground_beef": "绞牛肉", +  "item.gourmet.buns": "小圆面包", +  "item.gourmet.ice_cube": "冰块", +  "item.gourmet.carrot_slices": "胡萝卜片", + +  "fluid_type.gourmet.seed_oil": "籽油",  ⬇️ Suggested change -  "fluid_type.gourmet.seed_oil": "籽油", +  "fluid_type.gourmet.seed_oil": "种子油",  ⬇️ Suggested change -  "fluid_type.gourmet.seed_oil": "籽油", +  "fluid_type.gourmet.seed_oil": "植物油",  
籽油还真没听过，叫种子油的多（GTCE），植物油的也有点

**Comment by SouthWindmh** [comment]
如果有修改建议的话需要我重新pr或者重传文件吗

**Comment by seaseii** [comment]
> 如果有修改建议的话需要我重新pr或者重传文件吗

直接在本pr修改即可（

**Comment by seaseii** [comment]
> 其实我本来是想要翻译成法兰西吐司，但维基百科告诉我这玩意用的最多的名称是法国吐司。

法式/法国其实都可，看喜好了（

**Comment by SouthWindmh** [comment]
还有一个籽油瓶应该改成种子油瓶
有什么办法可以自己提交suggestion吗？(



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年8月7日(周一) 下午3:52
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] Create Gourmet 翻译提交 (PR #3624)




 
@seaseii commented on this pull request.
 
 
In projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:
 &gt; +  "item.gourmet.cheese_danish_dough": "奶酪丹麦酥面团", + +  "item.gourmet.dumpling_wrapper": "包子皮", +  "item.gourmet.incomplete_dumpling": "包子（半成品）", +  "item.gourmet.dumpling": "包子", + +  "item.gourmet.ice_cream_scoop": "冰淇淋球", +  "item.gourmet.chocolate_ice_cream_scoop": "巧克力冰淇淋球", +  "item.gourmet.sweet_berry_ice_cream_scoop": "甜浆果冰淇淋球", +  "item.gourmet.ice_cream_cone": "冰淇淋球甜筒", +  "item.gourmet.chocolate_ice_cream_cone": "巧克力冰淇淋甜筒", +  "item.gourmet.sweet_berry_ice_cream_cone": "甜浆果冰淇淋甜筒", +  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒", + +  "item.gourmet.carrot_cake": "胡萝卜蛋糕", +  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底",  ⬇️ Suggested change -  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底", +  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕胚",

**Comment by seaseii** [comment]
> 或者用 Github 内置的代码编辑器更改

但是很看脸，没七根木棍大概率进不去（其实有没有都差不多）
按键盘上的`句号`键就可以进入（或者将网页地址修改为github.dev/什么什么）

**Comment by SouthWindmh** [comment]
已经改好了，七木棍什么还是有的，不然我觉得我连github.com都上不去

**Comment by seaseii** [comment]
> 已经改好了，七木棍什么还是有的，不然我觉得我连github.com都上不去

雀食

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  "item.gourmet.sweet_berry_ice_cream_cone": "甜浆果冰淇淋甜筒",
+  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒",
+
+  "item.gourmet.carrot_cake": "胡萝卜蛋糕",
+  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕胚",
+
+  "item.gourmet.incomplete_carrot_cake": "胡萝卜蛋糕（半成品）",
+
+  "item.gourmet.variety_dessert_platter": "缤纷甜品拼盘",
+  "item.gourmet.comforting_dumpling_meal": "温馨包子餐",
```
Comment (Rinko1231) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:38]:
```suggestion
  "item.gourmet.comforting_dumpling_meal": "温馨包子套餐",
```

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  "item.gourmet.cooked_bacon": "熟培根",
+  "item.gourmet.caramelized_bacon": "焦糖培根",
+  "item.gourmet.raw_drumstick": "生鸡腿",
+  "item.gourmet.cooked_drumstick": "熟鸡腿",
+  "item.gourmet.fried_chicken": "炸鸡",
+  "item.gourmet.butter": "黄油",
+  "item.gourmet.cheese": "奶酪",
+  "item.gourmet.croissant": "牛角包",
+  "item.gourmet.croissant_dough": "牛角包面团",
+  "item.gourmet.incomplete_croissant_dough": "牛角包面团片",
```
Comment (Rinko1231) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:12]: 百科来看译为`可颂`的更多，而且也更顺口点

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒",
+
+  "item.gourmet.carrot_cake": "胡萝卜蛋糕",
+  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕胚",
+
+  "item.gourmet.incomplete_carrot_cake": "胡萝卜蛋糕（半成品）",
+
+  "item.gourmet.variety_dessert_platter": "缤纷甜品拼盘",
+  "item.gourmet.comforting_dumpling_meal": "温馨包子餐",
+  "item.gourmet.southern_breakfast": "南式早餐",
```
Comment (ChuijkYahus) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:39]:
```suggestion
  "item.gourmet.southern_breakfast": "南方早餐",
```
仅建议

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+  "item.gourmet.incomplete_breakfast_sandwich": "早餐三明治（半成品）",
+  "item.gourmet.breakfast_sandwich": "早餐三明治",
+  "item.gourmet.chicken_sandwich": "鸡肉三明治",
+  "item.gourmet.ice_cream_sandwich": "冰淇淋三明治",
+
+  "tooltip.gourmet.dream_roll": "它的味道会让你梦回故乡",
+
+  "message.gourmet.respawn_dimension_not_found": "未找到生成维度！",
+  "message.gourmet.no_spawn_point_set": "未设置出生点!",
```
Comment (ChuijkYahus) [inline projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json:80]:
```suggestion
  "message.gourmet.no_spawn_point_set": "未设置出生点！",
```

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json (hunk 6 above)
Comment (SouthWindmh) [reply projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json]: 已更改

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json (hunk 7 above)
Comment (SouthWindmh) [reply projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json]: 已更改

File: projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json (hunk 8 above)
Comment (SouthWindmh) [reply projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json]: 已更改

**Review by ChuijkYahus — APPROVED** [review]
感觉没啥大问题了

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**